__tests__/utils/StringUtils.test.js

```diff
@@ -27,12 +27,13 @@ describe('SringUtils should be run correctly', () => {
 
     const expiredHourse = moment().subtract('hours', 2);
     const expiredDays = moment().subtract('day', 2);
-
+    const minute = moment().add(40, 'minute');
     expect(getPollTime(hours)).toEqual('1h 59m left');
     expect(getPollTime(days)).toEqual('1d left');
     expect(getPollTime(minutes)).toEqual('1m left');
     expect(getPollTime(expiredDays)).toEqual('Poll closed 2d ago');
     expect(getPollTime(expiredHourse)).toEqual('Poll closed 0m ago');
+    expect(getPollTime(minute)).toEqual('39m left');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the variable name `expiredHourse`. It should be `expiredHours`. Also, the moment subtraction for hours is not correct. It should subtract 2 hours, not 'hours' which is not a valid number.
```diff
-    const expiredHourse = moment().subtract('hours', 2);
+    const expiredHours = moment().subtract(2, 'hours');
```
And then replace all instances of `expiredHourse` with `expiredHours`.
```diff
-    expect(getPollTime(expiredHourse)).toEqual('Poll closed 0m ago');
+    expect(getPollTime(expiredHours)).toEqual('Poll closed 2h ago');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->